docs/upgrade.md

```diff
@@ -91,6 +92,20 @@ def lambda_handler(event: DynamoDBStreamModel, context: LambdaContext):
 
 ```
 
+## Importing Pydantic objects
+
+We have stopped exporting Pydantic objects directly from `aws_lambda_powertools.utilities.parser.pydantic` to prevent potential confusion caused by Pydantic V2's significant changes in type, class, and method exports.
```

Comment:
```suggestion
You can now import Pydantic objects directly rather than from `aws_lambda_powertools.utilities.parser.pydantic`. We've made this change to streamline the import structure and allow you to use new Pydantic features as soon as they become available.
```

Reply:
I like the idea but the `and allow you to use new Pydantic features as soon as they become available.` part is not realistic. 

What about

> You can now import Pydantic objects directly rather than from aws_lambda_powertools.utilities.parser.pydantic. This change provides a more intentional import structure, allowing you to import only the specific Pydantic components you need.

Reply:
That's ok - but then I think the next sentence might be somewhat redundant

Reply:
What about this?

![image](https://github.com/user-attachments/assets/46fd88ef-aae6-4ddb-a352-1170db7e83a9)